blank_rust_project/contract/src/lib.rs

```diff
@@ -60,7 +60,7 @@ mod tests {
     fn set_get_message() {
         let context = get_context(vec![], false);
         testing_env!(context);
-        let mut contract = StatusMessage::default();
```

Comment:
this is technically an unrelated change. I can pull out into diff PR if we feel like that's needed for housekeeping.